### PR TITLE
IGAPP-378: Remove RTL workaround

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by maxammann — APPROVED** [review]

File: native/src/components/__tests__/NewsDetail.spec.tsx [hunk 1]
```diff
@@ -46,7 +48,7 @@ describe('NewsDetail', () => {
     fireEvent.press(getByText('mailto:app@integreat-app.de'))
     expect(navigateToLink).toHaveBeenCalledWith('mailto:app@integreat-app.de', language, 'mailto:app@integreat-app.de')
     fireEvent.press(getByText('https://integreat.app'))
-    expect(navigateToLink).toHaveBeenCalledWith('https://integreat.app', language, 'https://integreat.app')
+    expect(navigateToLink).toHaveBeenCalledWith('https://integreat.app/', language, 'https://integreat.app/')
```
[the remark this answers is not in the document]
Comment (Taggotty) [reply native/src/components/__tests__/NewsDetail.spec.tsx:51]: no. this is react-native-render-html stuff. But after some testing I found out that it only appends the slash to the domain and not the path:
https://integreat.app -> https://integreat.app/
https://integreat.app/testumgebung -> https://integreat.app/testumgebung

I guess they now normalize the path.

File: native/src/components/TimeStamp.tsx [hunk 2]
```diff
@@ -14,36 +14,38 @@ const TimeStampText = styled.Text`
...
+  showText = true,
+  format = 'LL'
+}: PropsType): ReactElement => {
+  const { i18n, t } = useTranslation('common')
+  // only show day, month and year
+  const dateText = formatter.format(lastUpdate, {
+    format
+  })
+  return (
+    <DirectionContainer theme={theme} language={i18n.language}>
```
Comment (Taggotty) [inline native/src/components/TimeStamp.tsx:44]: passing `i18n.language` will fix the alignment of Last Update, when system language is ltr and content language is rtl.

**Comment by klinzo** [comment]
Where can we test this?

**Review by klinzo — APPROVED** [review]
Tested on android, works nicely :) Last update also fixed...

File: native/src/components/TimeStamp.tsx (hunk 2 above)
Comment (klinzo) [reply native/src/components/TimeStamp.tsx:44]: @Taggotty could you check the news list as well please? Seems like we have the same problem there on native

Comment (Taggotty) [reply native/src/components/TimeStamp.tsx:44]: Hmmm. Cannot reproduce this. Does it only happen for local news?

Comment (klinzo) [reply native/src/components/TimeStamp.tsx:44]: Don't know, then just ignore it for now and I will look into it. Thx for checking :)

**Pull request merged**